test/enrollment_repository_test.rb

```diff
@@ -1,77 +1,91 @@
 require "minitest/autorun"
 require "minitest/pride"
 require "enrollment_repository"
-require "pry"
-require "enrollment"
 
 class EnrollmentRepositoryTest < Minitest::Test
-  def fixture_path  
-    File.expand_path("fixtures/Kindergartners in full-day program.csv", __dir__)
+  def kindergarten_participation_access_hash
+    path = File.expand_path("fixtures/Kindergartners in full-day program.csv", __dir__)
+    
+    { :kindergarten_participation => path }
+    
   end
 
-  def fixture_path_high_school
-    File.expand_path("fixtures/High school graduation rates.csv", __dir__)
+  def hs_graduation_rate_access_hash
+    path = File.expand_path("fixtures/High school graduation rates.csv", __dir__)
+
+    { :high_school_graduation_rates => path }
   end
 
-  def er
-    @er ||= begin
-      er = EnrollmentRepository.new
-      er.load_data({
-        :enrollment => {
-          :kindergarten => fixture_path,
-          :high_school_graduation => fixture_path_high_school
-        }
-      })
-      er
+  def setup
+    key_and_filepath_array_of_hashes = [kindergarten_participation_access_hash, hs_graduation_rate_access_hash]
+
+    @enrollment_repository ||= begin
+      enrollment_repository = EnrollmentRepository.new
+      enrollment_repository.load_enrollment_data(key_and_filepath_array_of_hashes)
+      enrollment_repository
     end
   end
+
+  def teardown
+    @enrollment_repository = nil
+  end
+
   def test_it_can_find_an_enrollment_object
-    assert_kind_of Enrollment, er.find_by_name("ACADEMY 20")
+    assert_kind_of Enrollment, @enrollment_repository.find_by_name("ACADEMY 20")
```

Comment:
Once again, instance variables created in our setup/teardown ensure that our tests are solid on their own.